support-workers/cloud-formation/src/templates/cfn-template.yaml

```diff
@@ -203,3 +203,75 @@ Resources:
       EvaluationPeriods: 1
       Statistic: Sum
     DependsOn: SupportWorkersPROD
+
+  NoPaypalContributionsInOneHourAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: CreateProdResources
+    Properties:
+      AlarmActions:
+        - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:contributions-dev
+      AlarmName: !Sub support-workers ${Stage} No successful recurring paypal contributions for an hour
+      MetricName: PaymentSuccess
+      Namespace: support-frontend
+      Dimensions:
+        - Name: PaymentProvider
+          Value: Paypal
+        - Name: ProductType
+          Value: Contribution
+        - Name: Stage
+          Value: ${Stage}
+      ComparisonOperator: LessThanOrEqualToThreshold
+      Threshold: 0
+      Period: 300
+      EvaluationPeriods: 12
+      Statistic: Sum
+      TreatMissingData: breaching
```

Comment:
this can be very confusing setting up an alarm.  if you have 12 evaluation periods, I have a feeling it will try hard to look back and find valid/nonmissing periods for longer than the 12 periods, and only if it can't find any data back for long enough it will go for breaching.  I think.  The documentation is very confusing though.

However if, you have copied this from the alarm that's working for single, then it must be right!

Reply:
I have indeed copied it from the payment-api alarm! It is a bit confusing though